tooling/build-and-use.mjs

```diff
@@ -6,7 +6,15 @@ import { stdin as input, stdout as output, } from 'node:process'
 
 async function runTasks() {
     console.clear()
-    console.log(chalk.bold.yellow('Running build and use tasks...\n'))
+    const presetApp = process.argv.at(2).slice(2)
```

Comment:
Maybe a bit nitpicky, but add check that arg starts with "--", if not, sending it withou "--" makes i.e. acquire -> "quire" with no direct error message that preset app isnt found